public/logo_transparent.svg

```diff
@@ -0,0 +1,12 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 600 600">
```

Comment:
Can you embed this in `logo.svg` so we only need to change 1 file to update the logo.